go-controller/pkg/networkmanager/network_controller.go

```diff
@@ -339,6 +339,10 @@ func (c *networkController) ensureNetwork(network util.MutableNetInfo) error {
 	// otherwise setup & start the new network controller
 	nc, err := c.cm.NewNetworkController(network)
 	if err != nil {
+		if errors.Is(err, ErrNetworkControllerTopologyNotManaged) {
```

Comment:
The `syncAll` method is still handling `ErrNetworkControllerTopologyNotManaged` error, i think that is not needed anymore.